app/controllers/popularProductEnquiry.js

```diff
@@ -0,0 +1,30 @@
+const models = require("../models");
+const PopularProductEnquiry = models.PopularProductEnquiry;
+const consumeError = require("../functions/consumeError");
+
+module.exports = {
+  async index(req, res) {
+    try {
+      let getAllPopularProductEnquiry = await PopularProductEnquiry.findAll({
+        where: {},
+      });
+      return getAllPopularProductEnquiry;
+    } catch (error) {
+      consumeError(error);
+    }
+  },
+  async create(req, res) {
```

Comment:
can we implement Validations here? @ChaitanyaV98

Reply:
we have already added validations separately in requests folder! @ShubhamJoshi-1994